### ISSUE TITLE
Organise data collection about gender 

### ISSUE THREAD

**Issue by lauradereynal** [MozillaFoundation/digital-skills-observatory#248]
We need more data to answer research questions about gender. 

Ideas: 
- organise a female only focus group in August, after the intervention 
- organise a specific interview (over the phone)
  ...

**Comment by lauradereynal** [comment]
@secretrobotron @JoashMango @DuncanGW 

I vote for trying out a focus group in Nairobi, and if the results are promising, then we can replicate in other regions. 
I also think that a focus group would help us design further interview questions.

Yes let do it in Nairobi then the outcome will assist us in doing the same in other areas

**Comment by secretrobotron** [comment]
> organise a female only focus group in August

Will that get us biased results? Maybe we should have 2 groups with the same material, one of them with male participants.

**Comment by secretrobotron** [comment]
> a focus group would help us design further interview questions

And interventions! 🐈 💥

**Comment by WinnieMakokha** [comment]
@LauraReynal i think i agree with @secretrobotron on doing FGDs for both male and female to help inform further interventions especially.

**Comment by lauradereynal** [comment]
Agreed with you both. 
Let's try to organize them for the first week of september. How does that sound?

**Comment by JoashMango** [comment]
Am okay with first week of September. We need to develop the FGD guide earlier so that we can do pretest of it. @LauraReynal

**Comment by lauradereynal** [comment]
@JoashMango I agree with developing the guide early in August.
Let's talk about it during our meeting tomorrow

**Comment by lauradereynal** [comment]
@secretrobotron we made good progress thanks to the questions we added on the interview 5. 

I suggest we use: 
- Interview 6 to do the same, 
- the next profiling work to do the same. 

We need to come up with new questions and activities for these 2 future opportunities. 

@WinnieMakokha Are you willing to help on this topic still ? We could organize a brainstorming together in Kenya next week.
